monitor/quotas.log

```diff
@@ -0,0 +1 @@
+TIMESTAMP	CPUS	PREEMPTIBLE_CPUS	IN_USE_ADDRESSES	DISKS_TOTAL_GB
```

Comment:
I guess as this file grows along the time, it will be much slower to parse this single big file in the future? Do you think it is easy to add data slicing to it? e.g. one log file per month.

Reply:
Yes, that is a good thought for the future